### ISSUE TITLE
Consistent $i18n API and usage example

### ISSUE THREAD

**Issue by sgimeno** [santhoshtr/vue-banana-i18n#22]
In the repository README, in the **Basic usage** section, the following code snippet is shown:
```html
<div id="app">
  <h1>{{ $i18n('hello_world') }}</h1>
  <h2 class='result'>{{ $i18n('search_results', 10) }}</h2>
  <div class='status'>{{ $i18n('profile_change_message', 'Alice', 'female') }}</h2>
</div>
```
However the syntax `{{ $i18n('profile_change_message', 'Alice', 'female') }}` seems unsupported and the test for this feature [tests for a different function signature](https://github.com/santhoshtr/vue-banana-i18n/blob/master/test/vue-banana.test.js#L191) `{{ $i18n('profile_change_message', [ 'Alice', 'female' ]) }}`.  Can someone clarify which is the intended API for the calls to `$i18n()`? 
Thank you!

**Comment by santhoshtr** [comment]
This is a bug. It should be consistant with banana-i18n and mediawiki i18n where the parameters can be passed as values(no need for array).

This should work, but currently it does not.
```
  <div class='status'>{{ $i18n('profile_change_message', 'Alice', 'female') }}</h2>
```

**Comment by santhoshtr** [comment]
Fixed in latest version 2.2.0
